app/utils/member_list.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import MemberListRow from 'app/components/custom_list/member_list_row';
+import {displayUsername} from 'service/utils/user_utils';
+
+export const loadingText = {
+    id: 'mobile.loading_members',
+    defaultMessage: 'Loading Members...'
+};
+
+export function createMembersSections(data) {
+    const sections = {};
+    data.forEach((d) => {
+        const name = d.username;
+        const sectionKey = name.substring(0, 1).toUpperCase();
+
+        if (!sections[sectionKey]) {
+            sections[sectionKey] = [];
+        }
+
+        sections[sectionKey].push(d);
+    });
+
+    return sections;
+}
+
+export function renderMemberRow(user, sectionId, rowId, preferences, theme, selectable, onPress, onSelect) {
```

Comment:
That's totally true but aince we have 3 different scenes that render the exact same thing I wanted to create this here so we dont need to write the same code 3 times, makes sense?